# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/dvdbackup/dev-cdrom.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- dvdbackup-0.4.2/man/dvdbackup.1.vanilla	2025-06-08 16:54:44.860767386 +0000
+++ dvdbackup-0.4.2/man/dvdbackup.1	2025-06-08 16:55:05.313000000 +0000
@@ -61,7 +61,7 @@
 .TP
 .B \-i DEVICE, \-\-input=DEVICE
 where DEVICE is your DVD device.  This switch only needs to be used if your DVD
-device node is not /dev/dvd
+device node is not /dev/cdrom
 .TP
 .B \-o DIRECTORY, \-\-output=DIRECTORY
 where DIRECTORY is your backup target.  If not given, the current working
@@ -131,7 +131,7 @@
 .TP
 .BI dvdbackup\ \-I
 gathers information about the DVD.
-/dev/dvd is the default device tried - you need
+/dev/cdrom is the default device tried - you need
 to use 
 .B -i
 if your device name is different.
--- dvdbackup-0.4.2/src/main.c.vanilla	2025-06-08 16:54:44.863959941 +0000
+++ dvdbackup-0.4.2/src/main.c	2025-06-08 16:55:41.264244641 +0000
@@ -93,7 +93,7 @@
 
 	printf(_("\
   -i, --input=DEVICE       where DEVICE is your DVD device\n\
-                           if not given /dev/dvd is used\n\
+                           if not given /dev/cdrom is used\n\
   -o, --output=DIRECTORY   where directory is your backup target\n\
                            if not given the current directory is used\n"));
 	printf(_("\
@@ -153,7 +153,7 @@
 	int return_code = 0;
 
 	/* DVD Video device */
-	char* dvd = "/dev/dvd";
+	char* dvd = "/dev/cdrom";
 
 	/* Temp switch helpers */
 	char* aspect_temp = NULL;
